app/src/main/java/org/glucosio/android/activity/AddCholesterolActivity.java

```diff
@@ -57,6 +56,8 @@
     private TextView HDLChoTextView;
     private int pagerPosition;
     private Runnable fabAnimationRunnable;
+    private long editId = 0;
```

Comment:
The editId and editing are coming from activity to activity. Should we extract common logic into class or data?